Say/Say.swift

```diff
@@ -37,7 +39,14 @@ class SayAPI: NSObject {
         self.voice = voice
         super.init()
     }
+    /*
+    var temp2 : NSSpeechSynthesizerDelegate
```

Comment:
I think 'temp2' is not good as a name of variable.
I've known that naming is difficult for every developer not familiar with English, but It is important thing for other developer who will read your code.

To give you a little hint, just lowercase first character of your object, excluding prefix.
if object class is NSSpeechSynthesizerDelegate, change it like this `var speechSynthesizerDelegate`

All this opinion is my thought, so I'd like you to consider happily